docs/src/equations.md

```diff
@@ -65,13 +65,16 @@ We make use of the following operators
 * ``\bar{\boldsymbol{u}} = \boldsymbol{u}_h + I_{c}(\boldsymbol{u}_v)``
 
 * ``\Phi = g z`` is the geopotential, where ``g`` is the gravitational acceleration rate and ``z`` is altitude above the mean sea level.
+* ``\boldsymbol{b} = - (\rho - \rho_h) / \rho \nabla \Phi`` is the reduced gravitational accleration
```

Comment:
```suggestion
* ``\boldsymbol{b} = - (\rho - \rho_h) / \rho \nabla \Phi`` is the reduced gravitational acceleration
```

Reply:
Also, what is `\nabla`? How is it defined discretely?

Reply:
The discretization is the same as the current momentum equation. I actually didn’t use `b` in the vertical momentum equation. Maybe we don’t need to define `b` here?